README.md

```diff
@@ -22,12 +22,23 @@ SETUP
         INSTALLED_APPS += ('django_summernote', )
 
 3. Add `django_summernote.urls` to `urls.py`.
-
-        urlpatterns = [
-            ...
-            url(r'^summernote/', include('django_summernote.urls')),
-            ...
-        ]
+     - For Django 1.x
+
+           urlpatterns = [
+               ...
+               url(r'^summernote/', include('django_summernote.urls')),
+               ...
+           ]
+        
+     - For Django 2.x
+     
+           from django.urls import re_path, include
+           # ...
+           urlpatterns = [
+               ...
+               re_path(r'^summernote/', include('django_summernote.urls')),
```

Comment:
[Official documentation](https://docs.djangoproject.com/en/2.1/ref/urls/#url) says it will be deprecated in the future releases.

I'm not sure if 'path()' works. Documentation claims re_path is the alias for 'url()'